### PR TITLE
add Satellite 6.11 and 6.12 to PR template

### PR DESCRIPTION
**Pull request by evgeni**
* [x] I am familiar with the [contributing](https://github.com/theforeman/foreman-documentation/blob/master/CONTRIBUTING.md) guidelines.

Please cherry-pick my commits into:

* [ ] Foreman 3.5/Katello 4.7
* [ ] Foreman 3.4/Katello 4.6
* [ ] Foreman 3.3/Katello 4.5
* [ ] Foreman 3.2/Katello 4.4
* [ ] Foreman 3.1/Katello 4.3
* For Foreman 3.0 or older, please create a separate PR.
* We do not accept PRs for Foreman 2.3 or older.

### CODE REVIEW

**Review by maximiliankolb — APPROVED** [review]

File: .github/PULL_REQUEST_TEMPLATE.md [hunk 1]
```diff
@@ -5,8 +5,8 @@ Please cherry-pick my commits into:
 
 * [ ] Foreman 3.5/Katello 4.7
```
Comment (adamlazik1) [inline .github/PULL_REQUEST_TEMPLATE.md:6]: Thanks for the PR! Should we add this as well?
```suggestion
* [ ] Foreman 3.5/Katello 4.7 (Satellite 6.13)
```

Comment (evgeni) [reply .github/PULL_REQUEST_TEMPLATE.md:6]: there is no such thing as Satellite 6.13 yet, so I refrained from doing it.

we could say "planned Satellite 6.13"?

Comment (adamlazik1) [reply .github/PULL_REQUEST_TEMPLATE.md:6]: Sure.

**Review by adamlazik1 — APPROVED** [review]
LGTM, failed html check is unrelated.

**Pull request merged**